tests/page-objects/infinite-scroller.ts

```diff
@@ -43,73 +45,54 @@ export class InfiniteScroller {
     this.firstItemTile = this.infiniteScrollerSectionContainer
       .locator('article')
       .first();
+
+    this.gridDisplayMode = this.displayStyleSelector.getByTestId('grid-button');
+    this.listDisplayMode = this.displayStyleSelector.getByTestId('list-detail-button');
+    this.compactDisplayMode = this.displayStyleSelector.getByTestId('list-compact-button')
   }
 
   async clickViewMode(viewModeLocator: LayoutViewModeLocator) {
     await this.displayStyleSelectorOptions.locator(viewModeLocator).click();
   }
 
-  async assertLayoutViewModeChange(viewMode: LayoutViewMode) {
-    switch (viewMode) {
-      case 'tile':
-        await expect(
-          this.displayStyleSelector.getByTestId('grid-button'),
-        ).toHaveClass('active');
-        expect(await expect(this.infiniteScroller).toHaveClass(/grid/));
-        return;
-      case 'list':
-        await expect(
-          this.displayStyleSelector.getByTestId('list-detail-button'),
-        ).toHaveClass('active');
-        expect(await expect(this.infiniteScroller).toHaveClass(/list-detail/));
-        return;
-      case 'compact':
-        await expect(
-          this.displayStyleSelector.getByTestId('list-compact-button'),
-        ).toHaveClass('active');
-        expect(await expect(this.infiniteScroller).toHaveClass(/list-compact/));
-        return;
-      default:
-        return;
-    }
+  async waitForFirstItemTile() {
+    const loadingTile = this.firstItemTile.locator('collection-browser-loading-tile');
+    await loadingTile.waitFor({ state: 'hidden' });
+    await this.firstItemTile.waitFor({ state: 'attached' });
+    await this.firstItemTile.waitFor({ state: 'visible' });
```

Comment:
Q: I'm curious, is it required that we wait for each of these in sequence? Is it not enough to just wait for the second one?